### DISCUSSION TITLE
cmdVel() can not use

### DISCUSSION THREAD

**Discussion by powehi2020** [USC-ACTLab/crazyswarm#764] [Q&A]
When i use cmdVel(), I also use rostopic echo /cf2/cmd_vel to monitor the order. It shows that some order has published to the drone, but the drone do not move, even if I have given thrust of 2^16 to the drone. I also tried to firstly send few messages such as 0 0 0, but the drone did not respond.
can anybody give me some advice? thk u!!

**Comment by jc-bao** [comment]
👍  The same issue here. No matter what I published with `cmd_vel()`, the drone's thrust becomes 0.

**Comment by jc-bao** [comment]
I have resolved the issue by following another issue's instructions. #264
